pom.xml

```diff
@@ -26,7 +26,11 @@
 
         <!--Maven plugins-->
         <groovy.version>2.5.15</groovy.version>
+        
+        <!--Minimum required maven version - 3.5.4 was released 2018-06-21 --> 
+        <maven.version.min>3.5.4</maven.version.min>
```

Comment:
Fever versions to support means fewer candidates for bugs. Currently 3.6.3 is the version used to release Cucumber so that would be preferable.